server/db/models/review.js

```diff
@@ -0,0 +1,22 @@
+var mongoose = require('mongoose');
+
+
+var ReviewSchema = new Schema({
+  stars: {
+    type: Number
+  },
+  description: {
+    type: String
+  },
+  wine: {
```

Comment:
moving this into the wine schema instead - wine has many reviews, they are only viewed on the wine page